plugins/modules/route53_health_check.py

```diff
@@ -86,6 +85,25 @@
       - Will default to C(3) if not specified on creation.
     choices: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ]
     type: int
+  health_check_name:
+    description:
+      - Name of the Health Check.
+      - Used together with I(use_unique_names) to set/make use of I(health_check_name) as a unique identifier.
+    type: str
+    required: False
+    aliases: ['name']
+  use_unique_names:
+    description:
+      - Used together with I(health_check_name) to set/make use of I(health_check_name) as a unique identifier.
+    type: bool
+    required: False
```

Comment:
When we want this to become the default behaviour (e.g. with community.aws 6), we need a warning that the behavior will change. `health_check_name` will become `required: True` and `use_unique_names` becomes `default: True`.